Portal/src/Datahub.Functions/AzureConfig.cs

```diff
@@ -84,4 +84,6 @@ public class AdoConfig
     public string OrgName { get; set; } = "DataSolutionsDonnees";
     public string ProjectName { get; set; } = "FSDH SSC";
     public string URL { get; set; } = "https://dev.azure.com/{organization}/{project}/_apis/wit/workitems/${workItemTypeName}?api-version=6.0";
+    public string GetPipelineURL { get; set; } = "https://dev.azure.com/{organization}/{project}/_apis/pipelines?api-version=7.1-preview.1";
```

Comment:
Can update these variable names to `WorkItemsUrlTemplate` and `ListPipelinesUrlTemplate`